packages/renderer/src/lib/onboarding/OnboardingItem.svelte

```diff
@@ -24,8 +29,15 @@ $: html;
 let configurationItems: IConfigurationPropertyRecordedSchema[];
 let configurationItem: IConfigurationPropertyRecordedSchema | undefined;
 $: configurationItem;
+let whenEvaluated: boolean | undefined;
 
 onMount(() => {
+  if (item.when) {
+    const when = normalizeOnboardingWhenClause(item.when, extension);
+    const whenDeserialized = ContextKeyExpr.deserialize(when);
+    whenEvaluated = whenDeserialized?.evaluate(getContext());
+  }
+
```

Comment:
I think one of the issue is that with onMount it's not reactive

Reply:
Yeah, I believe I encountered this issue too here: https://github.com/containers/podman-desktop/issues/4052

Do you suggest we subscribe to it / remove it from onMount?

Reply:
You should subscribe to the context as when something changes there you need to re-evaluate the when clause.

Reply:
I think you could evaluate it in Onboarding.svelte when looping on the content (OnboardingItem) to display.